commit f36ca7982f6bd5e9827219ed4f3c5a1fbf5d7bdf
Author: Jia Tan <jiat0218@gmail.com>
Date:   Sat May 13 21:21:54 2023 +0800

    liblzma: Slightly rewords lzma_str_list_filters() documentation.
    
    Reword "options required" to "supported options". The previous may have
    suggested that the options listed were all required anytime a filter is
    used for encoding or decoding. The reword makes this more clear that
    adjusting the options is optional.

diff --git a/src/liblzma/api/lzma/filter.h b/src/liblzma/api/lzma/filter.h
index fa612455..98b46df8 100644
--- a/src/liblzma/api/lzma/filter.h
+++ b/src/liblzma/api/lzma/filter.h
@@ -731,7 +731,7 @@ extern LZMA_API(lzma_ret) lzma_str_from_filters(
  *
  * If a filter_id is given then only one line is created which contains the
  * filter name. If LZMA_STR_ENCODER or LZMA_STR_DECODER is used then the
- * options required for encoding or decoding are listed on the same line too.
+ * supported options for encoding or decoding are listed on the same line too.
  *
  * If filter_id is LZMA_VLI_UNKNOWN then all supported .xz-compatible filters
  * are listed: